glyphs2fontir/src/source.rs

```diff
@@ -608,31 +608,17 @@ impl Work<Context, WorkId, WorkError> for FeatureWork {
     }
 }
 
-/// What side of the kern is this, in logical order
-enum KernSide {
-    Side1,
-    Side2,
-}
-
-impl KernSide {
-    fn class_prefix(&self) -> &'static str {
-        match self {
-            KernSide::Side1 => "@MMK_L_",
-            KernSide::Side2 => "@MMK_R_",
-        }
-    }
-
-    fn group_prefix(&self) -> &'static str {
-        match self {
-            KernSide::Side1 => "public.kern1.",
-            KernSide::Side2 => "public.kern2.",
-        }
-    }
+fn parse_kern_group(name: &str) -> Option<KernGroup> {
```

Comment:
could this be a method on KernGroup maybe?

Reply:
the issue is that the parsing behaviour depends on the backend, so it doesn't make sense to hang it off the common type.